platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/SimpleMapActivity.java

```diff
@@ -12,22 +14,30 @@
 public class SimpleMapActivity extends AppCompatActivity {
 
   private MapView mapView;
+  private MapboxMap mapboxMap;
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_map_simple);
     mapView = findViewById(R.id.mapView);
     mapView.onCreate(savedInstanceState);
-    mapView.getMapAsync(mapboxMap -> mapboxMap.setStyle(
-      new Style.Builder().fromUrl(Style.MAPBOX_STREETS)
-    ));
+    mapView.getMapAsync(map -> {
+      mapboxMap = map;
+      mapboxMap.setStyle(
+        new Style.Builder().fromUrl(Style.MAPBOX_STREETS)
+      );
+    });
   }
 
   @Override
   protected void onStart() {
     super.onStart();
     mapView.onStart();
+    if (mapboxMap != null) {
+      // Regression test for #14394
+      mapboxMap.queryRenderedFeatures(new PointF(0,0));
```

Comment:
How about moving the test to one of the query activities and leaving this one simple? :)

Reply:
atm, the query activities don't include a test for restarting them but that can be changed.

Reply:
I was just thinking about a different tetstapp acttivity